### DISCUSSION TITLE
Skeleton fill

### DISCUSSION THREAD

**Discussion by XantreDev** [nandorojo/moti#296] [Ideas]
Maybe there should exist other api for skeleton. Like `<ImageBackground />`, but in case of `Image` we can apply `StyleSheet.absoluteFill` to regular image, but in case of skeletons we can't. Current implementation of Skeleton wraps content into the `View` and it brakes flex layout, absolute positioning and so on.
I think solutions is to extract this code to separate component
![image](https://github.com/nandorojo/moti/assets/57757211/2bb00830-e1ad-4330-8423-4e9161b1527a)

**Comment by XantreDev** [comment]
I can provide more detailed code

**Reply by nandorojo** [reply]
you can provide your own absolute position view and put the skeleton inside of that with `width` and `height` props set to `100%`
